src/main/java/org/alfresco/repo/search/impl/solr/SolrQueryClient.java

```diff
@@ -38,7 +38,7 @@
  */
 public interface SolrQueryClient extends BeanFactoryAware, InitializingBean
 {
-    public ResultSet executeQuery(final BasicSearchParameters searchParameters, String language);
+    public ResultSet executeQuery(final SearchParameters searchParameters, String language);
```

Comment:
Will this cause backwards compatibility issues?

Reply:
No as this is a new interface and SearchParameter precedes BasicSearchParameters